src/module.ts

```diff
@@ -120,14 +124,23 @@ export default defineNuxtModule<ModuleOptions>({
       resolve('./runtime/registry'),
     ])
 
+    addImports({
+      from: resolve('./runtime/utils'),
+      name: 'useRegistryScript',
+    })
+
     addComponentsDir({
       path: resolve('./runtime/components'),
     })
 
     const scripts = registry(resolve)
+    scripts.push({
+      ...(await addGoogleTagManager()).registry,
+      ...(await addGoogleAnalytics()).registry,
```

Comment:
should we check if an entry with the same id exists and replace it? 
Seems safer, to avoid dupes (not sure how it could happen, but maybe good to safe-guard)

Reply:
Yeah, it's just while writing the PR. it's messy right now, i'll clean it before submiting it